omnibot/processor.py

```diff
@@ -370,7 +370,12 @@ def _handle_message_callback(message, callback):
             message.match_type,
             message.match
         ),
-        extra={**message.event_trace, 'callback': callback},
+        extra={
+            **message.event_trace,
+            'module': callback['module'],
+            'request_kwargs': callback.get('kwargs', {}).get('request_kwargs', {}),
+            'client_kwargs': callback.get('kwargs', {}).get('client_kwargs', {}).get('service', ""),
```

Comment:
i thought this was the potential leak?

Reply:
oh i guess this was meant to be
`'client_kwargs': {'service': callback.get(...).get(...).get('service', "")}`

Reply:
ah yeah could have nested it that way instead

Reply:
opened https://github.com/lyft/omnibot/pull/35 to fix